### ISSUE TITLE
Element passed to viewer has position style overridden

### ISSUE THREAD

**Issue by sv-matt-smith** [openseadragon/openseadragon#1301]
When instantiating a new viewer, if the 'element' option is used passing reference to an element, this element has it's position style overridden to "relative".

If the "id" option is used passing the id of this same element, then the position style is not affected.

I have created a [simple example](https://codepen.io/anon/pen/OjrmzV). (Simply un-comment the 'element' option in the constructor to see the change in behaviour. Note the element is initially positioned 'absolute').

I believe this is the line setting the style:
https://github.com/openseadragon/openseadragon/blob/master/src/controldock.js#L62

**Comment by iangilman** [comment]
Good catch. I imagine we're doing that so that the element can be a positioning parent to the things inside of it, but of course `absolute` does the same thing. It seems the fix would be to only use relative if the position is `static`. Would you be up for making this change?

**Comment by sv-matt-smith** [comment]
I'm happy to have a go at making the change. 

Are you concerned about the differing behaviour between the `element` and `id` option? I feel there may be more to this than restricting to `static` elements.

**Comment by iangilman** [comment]
Yes, I agree it should be the same. Investigating it all together sounds good to me!
